topiary-cli/src/main.rs

```diff
@@ -57,60 +57,47 @@ async fn run() -> CLIResult<()> {
             let (_, mut results) = async_scoped::TokioScope::scope_and_block(|scope| {
                 for input in inputs {
                     scope.spawn(async {
-                        let result: CLIResult<()> = match input {
-                            Ok(input) => {
-                                let language = cache.fetch(&input).await?;
-                                let output = OutputFile::try_from(&input)?;
```

Comment:
FYI this should work once the `try` block  is stabilized:
https://doc.rust-lang.org/nightly/unstable-book/language-features/try-blocks.html

```diff
-                           Ok(input) => {
+                           Ok(input) => try {
                                let language = cache.fetch(&input).await?;
                                let output = OutputFile::try_from(&input)?;
```